GraphQueryMessage query
---
byte FALSE = 0
byte TRUE = 1
byte QUERY_FAILED = 2

uint32 numberOfSolutionsFound
bool status
---
GraphAnswerMessage answer
byte status